app/services/mailgun_service.rb

```diff
@@ -0,0 +1,15 @@
+class MailgunService
+  class << self
+    def valid_post?(params)
+      signing_key = EnvironmentCredentials.dig(:mailgun, :webhook_signing_key)
```

Comment:
It'd be nice to have comments about which parts of this were specifically recommended by Mailgun, or which parts we chose for our reasons (and a link to their doc)